jib-gradle-plugin/build.gradle

```diff
@@ -72,6 +72,7 @@ pluginBundle {
 }
 
 gradlePlugin {
+  testSourceSets sourceSets.integrationTest, sourceSets.test
```

Comment:
Oh, we missed this? And I'm curious what the effect of adding this is?

Reply:
For some reason, in gradle 6.0+, without this, the gradleTestKit isn't applied to integration tests (so anything where we exectute gradle would probably not work anymore).